src/EditorFeatures/CSharpTest/Formatting/Indentation/SmartIndenterTests.cs

```diff
@@ -2570,6 +2570,26 @@ class C
                 expectedIndentation: 8);
         }
 
+        [Fact, Trait(Traits.Feature, Traits.Features.SmartIndent)]
+        public async Task DontCreateIndentOperationForBrokenBracketedArgumentList()
+        {
+            var code = @"
+class Program
+{
+    static void M()
+    {
+            string (userInput == ""Y"")
```

Comment:
@Pilchie The parser thinks that statement is a variable declaration statement with a bunch of missimg token, thats why it is a bracketed argument list. 

The repo doesnt need to be 8 spaces indented. The syntax structure is the only requirement.